src/components/utils/i18n.js

```diff
@@ -0,0 +1,32 @@
+import Polyglot from 'node-polyglot'
+
+import { locales, mobileLocales } from '../../../locales'
+import { isDesktop } from './'
+
+const defaultLocale = 'en'
+
+const unsupportedLocale = () => {
+  console.warn('Locale not supported')
+  return defaultLocale
+}
+
+const setLocale = (language) => {
+  if (!language || !language.locale) return defaultLocale
+  return locales[language.locale] ? language.locale : unsupportedLocale()
```

Comment:
This method is quite confusing:

- Consider merging this file with `locales/index.js`
- Make clear the difference between the locale, language, locales, and phrases objects. Do we need them all?

Reply:
Yes, good point on merging this file with `locales/index.js`. I will get rid of `language`. `locales` is the collection of translated strings, so I will probably rename that.